data/ruleset2.txt

```diff
@@ -0,0 +1,6 @@
+Winner,Loser,optionalIfTie
```

Comment:
These will need to match the new format:
White, Black, winner

Reply:
ohhhhh you merged something. i get it now lol